src/test/index.ts

```diff
@@ -0,0 +1,22 @@
+import { setupServer } from '../server';
+import { setupDatabase } from '../database';
+import axios from 'axios';
+import chai from 'chai';
+import * as dotenv from 'dotenv';
+
+describe('Query hello', () => {
+  before(async () => {
+    console.log('process.env.DATABASE_URL: ', process.env.DATABASE_URL);
+    dotenv.config({ path: __dirname + '/../../test.env' });
```

Comment:
```suggestion
    dotenv.config({ path: process.cwd() + '/test.env' });
```

O `process.cwd()` retorna a raiz do repo.